test/custom-error.js

```diff
@@ -0,0 +1,144 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/custom-error';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	},
+	parserOptions: {
+		sourceType: 'module'
+	}
+});
+
+ruleTester.run('custom-error', rule, {
+	valid: [
+		`
+			class FooError extends Error {
+				constructor(message) {
+					super(message);
+					this.name = 'FooError';
+				}
+			}
+		`,
+		`
+			class FooError extends Error {
+				constructor() {
+					super('My super awesome Foo Error');
+					this.name = 'FooError';
+				}
+			}
+		`
+	],
+	invalid: [
+		{
+			code: 'class FooError extends Error { }',
+			errors: [
+				{ruleId: 'custom-error', message: 'Add a constructor to your error'}
```

Comment:
Put all the messages in variables or an object, so they can be reused in all the invalid rules here.